warp10/src/main/java/io/warp10/continuum/gts/GTSHelper.java

```diff
@@ -1435,9 +1435,8 @@ public static final GeoTimeSerie subSerie(GeoTimeSerie gts, long starttimestamp,
     // Create sub GTS
     //
     
-    // The size hint impacts performance, choose it wisely...
     if (null == subgts) {
-      subgts = new GeoTimeSerie(128);
+      subgts = new GeoTimeSerie();
```

Comment:
The argument to the constructor only sets a hint for growing the GTS, it is probably better if this is still the case.
The constructor does not allocate the arrays.

Reply:
I am aware that it does not allocate the arrays. The reason I removed this is because it is the only place sizehint is set without any knowledge of how many data points could be added.